pkg/controller/snapshot_controller.go

```diff
@@ -651,16 +668,16 @@ func (ctrl *csiSnapshotController) createSnapshotOperation(snapshot *crdv1.Volum
 	}
 
 	if err != nil {
-		return nil, err
+		return nil, snapshotter.SnapshottingFinished, err
```

Comment:
should all of these be Background, since we now have a provider snapshot object, but haven't created a K8s object for it yet?

Reply:
Good point, updated all of these until SnapshottingInBackground